readme.markdown

```diff
@@ -21,7 +21,7 @@ readme resolves your module in the same way as `require()`
 
 > readme readme -g # for a globally installed module.
 
-> readme http -c    # for a core module
+> readme http      # for a core module
```

Comment:
There are some packages on npm that have the same name as a module in the Node core. Some are still used quite a bit (e.g. [punycode](https://www.npmjs.com/package/punycode)).

`-c` flag could help in resolving the potential ambiguity here. We could search locally first and then try to search in core (as it happens for global packages since #7).